build.gradle

```diff
@@ -139,6 +143,7 @@ sourceSets {
         }
         java {
             srcDir 'src/main'
+            srcDir '.gen'
```

Comment:
With idiomatic Java you pretty much never check in generated sources (unlike Go).

But we don't want to run the generator as part of the build step and want to eventually delete the generator entirely. At which point we'd want to put it in src/main/java with all the other sources.

I think it makes the most sense to put it in like `src/gen/java/` and then we merge it into `src/main/java` later.

The `.gen` directory pattern just seems really strange from a Java perspective.

Reply:
fixed